extensions/Cake.Xrm.SolutionPackager.yml

```diff
@@ -0,0 +1,16 @@
+Type: Addin
+Name: Cake.Xrm.SolutionPackager
+NuGet: Cake.Xrm.SolutionPackager
```

Comment:
Repo doesn't seem to be anywhere on GitHub... Blacklist

Reply:
Adding this addin to exclusion list: : https://github.com/cake-contrib/Cake.AddinDiscoverer/issues/194